### ISSUE TITLE
A11y_mssql for VSCode_mssql Extension_Message Window_ScreenReader: Screen Reader JAWS is not announcing Table summary for the table "Messages" on using letter "T"

### ISSUE THREAD

**Issue by prabhudevu** [microsoft/vscode-mssql#17618]
## **Actual:**
Screen Reader JAWS is not announcing Table summary for the table "Messages" on using letter "T". On using the letter "T" navigating to the table "Messages" no information is read.

### **Note:**

1. Issue is also repro for results table
2. Issue is no repro with NVDA Screen reader.

## **Expected:**
Screen Reader JAWS should announce the summary of the table- number of rows and number of columns and column headers on pressing "T" 

**GitHubTags:**#A11yTCS;#Win11;#DesktopApp;#BM_mssql for VSCode_Win32_Mar2023;#A11ySev2;#Benchmark;#mssql for VSCode;#ScreenReader;#JAWS;#FTP;#WCAG1.3.1;#A11yMAS;#Win32;#AINotInScope;#Element:Table;

## **Environment Details:**
Application Name: mssql for VSCode
Operating System: Win11[22H2]
OS Build: 22621.1413
Visual studio code insider: 1.77.0-insider (user setup)
Extension Version: v1.17.1
Screen reader: JAWS[2023.2210.29 ILM]

## **Repro Steps:**

1. Open Visual studio code
2. Tab till extension button and search for "mssql".
3. Tab till install and move to the SQL Server button.
4. Tab till Add Connection Button and insert the valid credentials.
5. Tab till the database and hit enter.
6. Tab till the System database and hit enter.
7. Tab till the Tables and hit enter.
8. Tab till any table under System tables and hit shift + F10
9. Tab till Select Top 1000 and hit enter
10. Tab till Execute Query and hit enter
11. Tab till "Messgaes" dropdown and hit enter
12. Turn on the screen reader JAWS
13. Press "T"

## **User Impact:**
Screen reader users think there are no tables present if nothing is being announced on pressing "T"

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

## **Attachment**
![17618_A11y_mssql for VSCode_mssql Extension_Message Window_ScreenReader](https://user-images.githubusercontent.com/98821792/225620364-e88bbf75-cd01-4169-808f-d05f3c9991c7.gif)

![17618_A11y_mssql for VSCode_mssql Extension_Message Window_ScreenReader](https://user-images.githubusercontent.com/98821792/225625704-7d2303fd-34be-4247-ae74-9a44942a79cc.gif)

**Comment by Roopesh-T** [comment]
Issue not repro in Version: 1.80.0-insider (user setup)
![17618_closed](https://github.com/microsoft/vscode-mssql/assets/86601127/d1f49d09-67da-423c-8ddc-bb19e810f96d)
** GitTags**: #closed;#regressed:06-22-23;
